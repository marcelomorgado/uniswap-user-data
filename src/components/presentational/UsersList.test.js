import React from "react";
import { shallow } from "enzyme";
import UsersList from "./UsersList";

it("renders the component", () => {
  const props = {
    hasNextPage: false,
    isNextPageLoading: false,
    loadNextPage: () => {},
    items: [],
    onTransactionsClick: () => {},
  };

  expect(shallow(<UsersList {...props} />)).toMatchSnapshot();
});
