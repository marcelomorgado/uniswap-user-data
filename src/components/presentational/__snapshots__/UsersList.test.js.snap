// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the component 1`] = `
<InfiniteList
  hasNextPage={false}
  isNextPageLoading={false}
  items={Array []}
  loadNextPage={[Function]}
  rowHeight={200}
  rowRenderer={[Function]}
/>
`;
